src/main/interface/cli.c

```diff
@@ -489,12 +489,17 @@ static void cliPrintVarRange(const clivalue_t *var)
     case (MODE_LOOKUP): {
         const lookupTableEntry_t *tableEntry = &lookupTables[var->config.lookup.tableIndex];
         cliPrint("Allowed values:");
+
+        bool firstVisibleEntry = true;
```

Comment:
This is to fix the space after the entry? I overlooked that, but I'd rather go for:
```
 for (uint32_t i = 0; i < tableEntry->valueCount; i++) {
￼              if (tableEntry->values[i]) {
                cliPrintf("%s", tableEntry->values[i]);
                if (i + 1 < tableEntry->valueCount) {
                    cliPrint(", ");
￼                  }
```

and add the first space after the `Allowed values:` above.

(Which makes it follow the flow of natural language.)

Reply:
To fix comma after colon if first entry is missing IIRC